programs/eosio-blocklog/main.cpp

```diff
@@ -262,7 +262,7 @@ void smoke_test(bfs::path block_dir) {
    for (uint32_t n = td.first_block; ; n += delta) {
       if (n > td.last_block)
          n = td.last_block;
-      td.find_block_pos(n);                                 //check block 'n' is where blocks.index says
+      td.block_pos(n);                                 //check block 'n' is where blocks.index says
```

Comment:
Is this the only reason why the extra code at the end of `block_pos` is needed? It seems like that extra code could just go here.